core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -221,6 +222,7 @@ public void runInContext() {
       return;
     }
 
+    // TODO(zdapeng): remove this after migration to interceptor based config selector.
     if (configSelector != null) {
       PickSubchannelArgs args = new PickSubchannelArgsImpl(method, headers, callOptions);
       InternalConfigSelector.Result result = configSelector.selectConfig(args);
```

Comment:
I'm concerned about this double-picking and it assuming we get the same results both times. Could you either migrate the current user of the `committedCallback` or convert the callback to an interceptor in `ConfigSelectingClientCall`? It seems that should be really easy. If not, then maybe disable this flow if the CallOptions contain the service config.

Reply:
I converted the callback to an interceptor for the moment.  (It seems not working to disable this flow if the CallOptions contain the service config)